src/libaktualizr/storage/sqlstorage_base.cc

```diff
@@ -77,6 +77,42 @@ bool SQLStorageBase::dbMigrateForward(int version_from, int version_to) {
   }
 
   for (int32_t k = version_from + 1; k <= version_to; k++) {
+    if (k == 18) {  // Hack, because we can't write to the file from sql scripts.
```

Comment:
Unfortunately we'll also need a backward migration hack as well, right?

Reply:
Unfortunatelly it is the biggest problem. The old aktualizr will not have this code and it is not possible to backmigrate this migration. We should store this migration in the database but we could not do this using sql.

Reply:
I think we should not migrate these files, the primary will download them again if needed anyway (or it should!)

Reply:
Fair points. Perhaps we should state that in the code as a comment, though.